tests/other_expectations/expect_column_values_to_not_be_null.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Will you also include the test for `expect_column_values_to_be_null` in this section, please?

Reply:
yes, see above.